History.md

```diff
@@ -1,4 +1,4 @@
-## main
+## 1.1.0 / 2020-06-24
```

Comment:
:thinking: `2.0.0` or `1.1.0`?

Reply:
`2.0.0`.  I also screwed up the date :) . Both fixed now.